### ISSUE TITLE
For not yet completed KYC go to the last not yet finished step

### ISSUE THREAD

**Issue by desfero** [Neufund/platform-frontend#3817]
**Background**
At the moment in the case, KYC was just started but not submitted we start the flow always from the beginning (choosing KYC request type). It would be good for the user experience to redirect to the last not yet finished step so guide the user and increase KYC success rate.

**Designs**
We don't do any designs changes here

**Acceptance criteria**
- [ ] Given user KYC verification flow is not yet completed when he clicks on `Continue verification flow` in the profile page then it should redirect to the last not yet finished KYC step.

**Technical notes**

**Testing notes**

**Comment by pstroinski** [comment]
Please comment how it should behave

**Comment by andywalton** [comment]
I agree with @desfero comment. It would be good for the user experience to redirect to the last not yet finished step if they continue KYC via their Profile view.
